apis/apis.json

```diff
@@ -299,6 +299,24 @@
         "Billing"
       ]
     },
+    {
+      "id": "Google.Cloud.BinaryAuthorization.V1Beta1",
+      "version": "1.0.0-beta01",
+      "type": "grpc",
+      "productName": "Binary Authorization API",
+      "productUrl": "https://cloud.google.com/binary-authorization/docs/reference/rpc",
+      "description": "The management interface for Binary Authorization, a system providing policy control for images deployed to Kubernetes Engine clusters.",
```

Comment:
Often this is phrased "Recommended Google client library..." - should this also be phrased this way?

Reply:
Yes, good call - will fix in the various commits.

Reply:
Fixed but then pushed to the wrong branch. So beta01 will have the suboptimal description - I'll create a new PR just to fix it for future releases.